etc/di.xml

```diff
@@ -105,4 +105,24 @@
             <argument name="customerSession" xsi:type="object">Magento\Customer\Model\Session\Proxy</argument>
         </arguments>
     </type>
```

Comment:
Should we define the default product service using `preferenceFor` tag? https://github.com/Nosto/nosto-magento2/blob/develop/etc/di.xml#L39